packages/components/psammead-social-embed/CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # Social Embed Changelog
 
-| Version       | Description                                                                                       |
-| ------------- | ------------------------------------------------------------------------------------------------- |
-| 1.0.2 | [PR#3397](https://github.com/bbc/psammead/pull/3397) Talos - Bump Dependencies - @bbc/psammead-styles |
-| 1.0.1 | [PR#3388](https://github.com/bbc/psammead/pull/3388) Update react-helmet to 6.0.0 |
-| 1.0.0 | [PR#3349](https://github.com/bbc/psammead/pull/3349) Resolve a11y issues and release version one. |
-| 0.1.0-alpha.4 | [PR#3319](https://github.com/bbc/psammead/pull/3319) Remove dependency on @bbc/psammead-oembed.   |
-| 0.1.0-alpha.3 | [PR#3311](https://github.com/bbc/psammead/pull/3311) Transpile with @loadable/babel-plugin.       |
-| 0.1.0-alpha.2 | [PR#3298](https://github.com/bbc/psammead/pull/3298) Add webpackChunkName to dynamic import.      |
-| 0.1.0-alpha.1 | [PR#3217](https://github.com/bbc/psammead/pull/3217) Initial creation of package.                 |
+| Version       | Description                                                                                           |
+| ------------- | ----------------------------------------------------------------------------------------------------- |
+| 1.0.3         | [PR?]() Add CSS styles for AMP                                                                        |
```

Comment:
- Oops, we're missing the PR information here.
- Also the description should probably say something like: `Update Canonical provider styles`